stonesoup/types/mixture.py

```diff
@@ -0,0 +1,96 @@
+from collections.abc import Sized, Iterable, Container
+
+from .base import Type
+from ..base import Property
+from .state import TaggedWeightedGaussianState, WeightedGaussianState
+
+
+class GaussianMixtureState(Type, Sized, Iterable, Container):
+    """
+    Gaussian Mixture State type
+
+    Represents the target space through a Gaussian Mxture. Individual Gaussian
+    components are contained in a :class:`list` of
+    :class:`WeightedGaussianState`.
+    """
+
+    components = Property(
+        [WeightedGaussianState],
+        default=[],
+        doc="""The initial list of :class:`WeightedGaussianState` components.
+        Default `None` which initialises with empty list.""")
+
+    current_tag = Property(
```

Comment:
So I would remove this, and instead use UUIDs. This would remove the need to manage the tags, as ID's will always be unique.